### PR TITLE
Add camel bindings

### LINKED ISSUE #3099: Add bindings for Camel model API
Model page here: https://huggingface.co/Writer/camel-5b-hf

### PR DESCRIPTION
**Pull request by heathexer**
Resolves #3099

### CODE REVIEW

File: tests/integration_tests/llms/test_camel.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""Test Camel API wrapper."""
+
+from langchain.llms.camel import Camel
+
+
+def test_camel_call() -> None:
+    """Test valid call to Camel API."""
+    llm = Camel()
+    output = llm("Say foo:")
+    assert isinstance(output, str)
```
Comment (vowelparrot) [inline tests/integration_tests/llms/test_camel.py:10]: Can we check the length of the output.strip() > 0 as well?

**Comment by vowelparrot** [comment]
Apart from using `requests` directly, how needed is this vs. the `HuggingFaceHub` LLM?

**Comment by hwchase17** [comment]
> Apart from using `requests` directly, how needed is this vs. the `HuggingFaceHub` LLM? If it's explicitly a HuggingfaceInferenceEndpoint, we should be reusing code

+1 to this, could this just use the huggingface integratoins under the hood?

if it can, we can still highlight by adding a section in the huggingface notebook about how to use this model

**Comment by vowelparrot** [comment]
@heathexer I opened an example #3187

**Comment by hwchase17** [comment]
closing and just gonna use examples in other nb

**Pull request closed without merging**